slack_sdk/audit_logs/v1/async_client.py

```diff
@@ -250,25 +248,112 @@ async def _perform_http_request(
                 "ssl": self.ssl,
                 "proxy": self.proxy,
             }
-            async with session.request(http_verb, url, **request_kwargs) as res:
-                response_body = {}
-                try:
-                    response_body = await res.text()
-                except aiohttp.ContentTypeError:
+            retry_request = RetryHttpRequest(
+                method=http_verb,
+                url=url,
+                headers=headers,
+                body_params=body_params,
+            )
+
+            retry_state = RetryState()
+            counter_for_safety = 0
+            while counter_for_safety < 100:
```

Comment:
We may want to remove this counter for simplicity. `while True` here should be safe enough as `retry_state.next_attempt_requested` is usually False